src/main/java/me/krumka/onlinebookshop/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package me.krumka.onlinebookshop.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import me.krumka.onlinebookshop.dto.BookDto;
+import me.krumka.onlinebookshop.dto.CreateBookRequestDto;
+import me.krumka.onlinebookshop.service.BookService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties
```suggestion
@RequestMapping("/books")
```